youtube_dlc/postprocessor/embedthumbnail.py

```diff
@@ -83,47 +93,82 @@ def is_webp(path):
             self.to_screen('Adding thumbnail to "%s"' % filename)
             self.run_ffmpeg_multiple_files([filename, thumbnail_filename], temp_filename, options)
 
-        elif info['ext'] == 'mkv':
+        elif info['ext'] in ['mkv', 'mka']:
+            # streams = self.get_metadata_object(filename)['streams']
             options = [
                 '-c', 'copy', '-map', '0', '-dn', '-attach', thumbnail_filename,
```

Comment:
Will the inclusion of `-dn` here remove the info.json attachment added in the add-metadata postprocessor?

Reply:
no. tested